Adding support for Syncthing Devices (#46660)
home-assistant/home-assistant.github.io: source/_integrations/syncthing.markdown

  
## Integration entities

> 

Reviewer: ```suggestion
## Supported functionality

The **Syncthing** integration provides the following entities.

### Sensors

```
I realize this section title is outside the scope of this PR. Changing it to standardize the structure, following the template:
https://github.com/home-assistant/home-assistant.io/blob/current/source/_integrations/_integration_docs_template.markdown?plain=1#L91

> The Syncthing integration adds a sensor for each configured device and each synced folder:

Reviewer: ```suggestion
The Syncthing integration adds a sensor for each configured device and each synced folder:

- One sensor per synced folder. Its state reflects the folder's synchronization status reported by Syncthing. Details such as the number of files and the folder size are available as state attributes.
- One sensor per configured device. Its state reflects the device's connection status: `online`, `connected`, `disconnected`, `paused`, or `unknown`. The entry representing your own Syncthing server always shows `online`. Details, such as the device ID and addresses, are available as state attributes.
```
Maybe add a brief description.

![Syncthing Sensors](/images/integrations/syncthing/sensors.png)

![Syncthing Device](/images/integrations/syncthing/device.png)

![Syncthing Folder](/images/integrations/syncthing/folder.png)

## Notes
Device Sensors are updated via events from Syncthing.

> Status may be outdated or set as `unknown` until the next event is generated by Syncthing.

Reviewer: ```suggestion
## Data updates

Device sensors are updated by events sent by Syncthing and refreshed via {% term polling %} every 120 seconds. A device sensor's status may be outdated or shown as `unknown` until the next event is received or the next poll completes.
```
https://github.com/home-assistant/home-assistant.io/blob/current/source/_integrations/_integration_docs_template.markdown?plain=1#L189
